src/Leanplum.ts

```diff
@@ -287,54 +289,58 @@ Use "npm update leanplum-sdk" or go to https://docs.leanplum.com/reference#javas
       callback = userAttributes
       userAttributes = {}
     }
+
     LeanplumRequest.userId = userId
+
     if (callback) {
       Leanplum.addStartResponseHandler(callback)
     }
 
-    InternalState.hasStarted = true
-    InternalState.startSuccessful = true
-    if (InternalState.devMode) {
-      Leanplum._lpSocket.connect(Leanplum._varCache)
+    Leanplum._internalState.hasStarted = true
+    Leanplum._internalState.startSuccessful = true
+
+    if (Leanplum._internalState.devMode) {
+      Leanplum._lpSocket.connect(Leanplum._varCache, Leanplum.createRequest)
     }
+
     Leanplum._varCache.loadDiffs()
-    InternalState.triggerStartHandlers()
+    Leanplum._internalState.triggerStartHandlers()
   }
 
   static stop() {
     // noinspection Annotator
-      LeanplumRequest.request(Constants.METHODS.STOP, undefined, {
-      sendNow: true,
```

Comment:
`sendNow: true` is missing after the refactoring, and in its place there's the devMode parameter which `createRequest` always sets? This will work in the same way in development mode, but will not send the request immediately in production.

Reply:
Updated, thanks!